source/_posts/2018-01-04-bad-design-is-bad-for-your-health-why-data-visualization-details-matter.markdown

```diff
@@ -0,0 +1,88 @@
+
+Context
+
+Once there is a truthful and clear foundation for data, the following techniques can bring users quicker insights and increase their confidence in data interpretation.
+
+If there is a normal, expected, or ideal range of values for the data, adding a target range is a simple but powerful way to add context. A light shaded region behind the grid lines can allow users to easily identify high and low values. Similar to a target range, a goal line can be added to indicate a discrete value that the plotted data should be achieving. A goal line and target range can be used together as well.
+
+It’s important to mark visualizations and dashboards with the date it was published. I also recommend including the date or date range of the data as well in order to maintain trust and relevancy of your visualization.
+
+Annotations are another simple way to provide your user with context. On-canvas notes and labels should be used sparingly, but can help highlight insights. These can be placed in context of a specific data point, a date or timeframe, etc.
+
+Creativity
+
+Once we have created a clear visualization and provided our user with additional context, we are ready to polish our design. There is great potential for creativity in data visualization, but it can cause more harm than good if done poorly. Let’s look at visual encoding methods, colors, and other visual design details that can enhance your visualization or dashboard.
+
+Qualitative and quantitative data is visually encoded, or translated, through a variety of graphical methods that make up each visualization type. [Research](https://ils.unc.edu/courses/2015_spring/inls541_001/Readings/Cleveland%20and%20McGill%201985%20-%20Graphical%20Perception%20and%20Cleveland1985-Graphical%20Methods%20for%20Analyzing%20Scientific%20Data.pdf) has been done on these visual encoding methods, giving us insights for human graphical perception. Statistician William Cleveland and his colleague, Robert McGill, tested the visual encoding methods and ranked them by accuracy of perception. [Their findings](https://ils.unc.edu/courses/2015_spring/inls541_001/Readings/Cleveland%20and%20McGill%201985%20-%20Graphical%20Perception%20and%20Cleveland1985-Graphical%20Methods%20for%20Analyzing%20Scientific%20Data.pdf) give us a scientific foundation for data visualization.
```

Comment:
The reference to the research link exists twice here, probably only need to include it once.

Reply:
Fixed, thanks :)